Dockerfile

```diff
@@ -1,25 +1,77 @@
-# from https://hub.docker.com/_/ubuntu/tags?page=1&name=22.04
-FROM ubuntu:22.04@sha256:bcc511d82482900604524a8e8d64bf4c53b2461868dac55f4d04d660e61983cb
+FROM ghcr.io/actions/actions-runner:2.313.0 AS base
+USER root
```

Comment:
do we need to change user later? I see the old version chmod to `github` user

Reply:
Chaning user permission is mandatory (and best practice) because built-in sh scripts do checks about user permissions ([here](https://github.com/actions/runner/blob/d296014f99f9926d132cd7de6ddb1352800dc222/src/Misc/layoutbin/installdependencies.sh#L5) and [here](https://github.com/actions/runner/blob/d296014f99f9926d132cd7de6ddb1352800dc222/src/Misc/layoutroot/config.sh#L6)).
`runner` user instead is already created in the base image with properly permissions and group assignments [here](https://github.com/actions/runner/blob/d296014f99f9926d132cd7de6ddb1352800dc222/images/Dockerfile#L48)